src/chains.ts

```diff
@@ -0,0 +1,49 @@
+export enum ChainId {
+  MAINNET = 1,
+  GOERLI = 5,
+  SEPOLIA = 11155111,
+  OPTIMISM = 10,
+  OPTIMISM_GOERLI = 420,
+  ARBITRUM_ONE = 42161,
+  ARBITRUM_GOERLI = 421613,
+  POLYGON = 137,
+  POLYGON_MUMBAI = 80001,
+  CELO = 42220,
+  CELO_ALFAJORES = 44787,
+  GNOSIS = 100,
+  MOONBEAM = 1284,
+  BNB = 56,
+  AVALANCHE = 43114,
+  BASE_GOERLI = 84531,
+  BASE = 8453
+}
+
+export const SUPPORTED_CHAINS = [
+  ChainId.MAINNET,
+  ChainId.OPTIMISM,
+  ChainId.OPTIMISM_GOERLI,
+  ChainId.ARBITRUM_ONE,
+  ChainId.ARBITRUM_GOERLI,
+  ChainId.POLYGON,
+  ChainId.POLYGON_MUMBAI,
+  ChainId.GOERLI,
+  ChainId.SEPOLIA,
+  ChainId.CELO_ALFAJORES,
+  ChainId.CELO,
+  ChainId.BNB,
+  ChainId.AVALANCHE,
+  ChainId.BASE,
+  ChainId.BASE_GOERLI
+] as const
+export type SupportedChainsType = typeof SUPPORTED_CHAINS[number]
+
+export enum NativeCurrencyName {
+  // Strings match input for CLI
+  ETHER = 'ETH',
+  MATIC = 'MATIC',
+  CELO = 'CELO',
+  GNOSIS = 'XDAI',
+  MOONBEAM = 'GLMR',
+  BNB = 'BNB',
+  AVAX = 'AVAX'
+}
```

Comment:
![README.md](https://github.com/Dirtyyt66/sdk-core/files/13718677/README.md)

![Not .txt](https://github.com/Dirtyyt66/sdk-core/files/13718680/Not.txt)

![README.md](https://github.com/Dirtyyt66/sdk-core/files/13718688/README.md)

![README.md](https://github.com/Dirtyyt66/sdk-core/files/13718691/README.md)

![bulk-sponsorships-template.xlsx](https://github.com/Dirtyyt66/sdk-core/files/13718699/bulk-sponsorships-template.xlsx)

![Not .txt](https://github.com/Dirtyyt66/sdk-core/files/13718710/Not.txt)
![README.md](https://github.com/Dirtyyt66/sdk-core/files/13718713/README.md)

![README.md](https://github.com/Dirtyyt66/sdk-core/files/13718716/README.md)

![README.md](https://github.com/Dirtyyt66/sdk-core/files/13718728/README.md)

K7JW